javascript/javascript1/week3/script1.js

```diff
@@ -23,3 +23,22 @@ names.splice(indexOfName, 1);
 
 console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
 
+//Exe-2 When will we be there??
+
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+};
+function notThisFunctionName(travelInformation) {
+    const time = travelInformation.destinationDistance / travelInformation.speed;
+    console.log(time);
+    let hours = Math.floor(time);
+    console.log(hours);
+    let minutes = Math.floor((time - hours) * 60);
+    return hours + " Hours " + minutes + " Minutes";
+
+}
+
+const travelTime = notThisFunctionName(travelInformation);
```

Comment:
I would change the name of the function :) notThisFunctionName

Reply:
Yes I would like to follow the commands(HYF- hw commands)